figures/balancer-id-sequence.mmd

```diff
@@ -0,0 +1,9 @@
+sequenceDiagram
+    participant Balancer
+    participant Monolith
+
+    note over Balancer, Monolith: Connection is already initialized
+    Balancer->>Balancer: Generates BalancerID on startup
+    Monolith->>Balancer: Request is received by Balancer.
+    Balancer->>Monolith: Balancer loads a room and sends BalancerID
```

Comment:
Why don't we just add a `B2MInit` message so we can just send the id once when the connection is established?

This would probably be better if you just modify `monolith-id-sequence` instead

Reply:
Yeah, that was the other thing I was thinking about. I'll change that now.